include/engine/guidance/assemble_steps.hpp

```diff
@@ -119,6 +120,8 @@ inline std::vector<RouteStep> assembleSteps(const datafacade::BaseDataFacade &fa
                 const auto exits = facade.GetExitsForID(step_name_id);
                 const auto distance = leg_geometry.segment_distances[segment_index];
                 auto classes = facade.GetClasses(path_point.classes);
+                // intersections contain the classes of exiting road
+                intersection.classes = std::move(classes);
```

Comment:
just a personal preference: I'd prefer not assigning a name to `classes` if we only use it to transfer the data. i.e. `intersection.classes = face.GetClasses()`.